### PR TITLE
Update codelab.md

### PR DESCRIPTION
**Pull request by pedrocarlos-ti**
Property title don't exists in Scaffold, replaced with appBar property.

### CODE REVIEW

**Comment by pedrocarlos-ti** [comment]
I signed it!

**Comment by johnpryan** [comment]
can you take a look at the merge conflicts?

File: src/docs/get-started/codelab.md [hunk 1]
```diff
@@ -243,8 +243,8 @@ packages, on [pub.dev][].
          return MaterialApp(
            title: 'Welcome to Flutter',
            home: Scaffold(
-    @@ -16,7 +18,7 @@
-               title: Text('Welcome to Flutter'),
+               appBar: AppBar(
```
Comment (johnpryan) [inline src/docs/get-started/codelab.md]: I think this line might be prefixed with 2 extra spaces. Can you make sure this is formatted correctly?

Comment (pedrocarlos-ti) [reply src/docs/get-started/codelab.md]: Yep

**Comment by johnpryan** [comment]
merged in this commit: https://github.com/flutter/website/commit/f20bcf11c030902f0565b569b74933a26f1e6c0c

Thanks for your contribution!

**Pull request closed without merging**